invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/PatronProfile/PatronPendingLoans/PatronPendingLoans.js

```diff
@@ -0,0 +1,174 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import {
+  Loader,
+  Error,
+  Pagination,
+} from '../../../../common/components';
+import {invenioConfig} from '../../../../common/config';
+import {
+  Container,
+  Grid,
+  Header,
+  Icon,
+  Item,
+  Label, Message,
+  Popup
+} from "semantic-ui-react";
+import isEmpty from 'lodash/isEmpty';
+import {getCover} from "../../config";
+import {Link} from "react-router-dom";
+import {FrontSiteRoutes} from "../../../../routes/urls";
+import {DocumentAuthors} from "../../components/Document";
+import {toShortDate} from "../../../../common/api/date";
+import {ILSItemPlaceholder} from "../../../../common/components/ILSPlaceholder/ILSPlaceholder";
+
+
+class LoanRequestListEntry extends Component {
+
+  render() {
+    const {loan} = this.props;
+    return (
+      <Item key={loan.metadata.pid} data-test={loan.metadata.pid}>
+        <Item.Image size='mini'
+                    src={getCover(loan.metadata.document_pid)}
+                    as={Link}
+                    disabled
+                    to={FrontSiteRoutes.documentDetailsFor(
+                      loan.metadata.document_pid)}
+        />
+
+        <Item.Content>
+          <Item.Header as={Link}
+                       to={FrontSiteRoutes.documentDetailsFor(
+                         loan.metadata.document_pid)}>
+            {loan.metadata.document.title}</Item.Header>
```

Comment:
For me, this type of formatting is a bit tougher to read. Isn't not the same as we've used in other components, is it?

Reply:
I'll fix the formatting